delphi-frontend/src/main/java/au/com/integradev/delphi/cfg/api/UnknownException.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Sonar Delphi Plugin
+ * Copyright (C) 2024 Integrated Application Development
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package au.com.integradev.delphi.cfg.api;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public interface UnknownException extends Successors {
```

Comment:
Can you explain the name of this `Successors` type?
I don't really get it at a glance (javadoc might help).

Reply:
I will add a javadoc.
High level though, if a block can raise an exception but the type of which is unknown, that is when this block occurs.
It just means that there are multiple potential successors, rather than a successor specifically directed to an arm of a try-catch-finally.